suite-common/redux-utils/src/index.ts

```diff
@@ -7,3 +7,5 @@ export * from './createReducerWithExtraDeps';
 export * from './createActionWithExtraDeps';
 export * from './createSingleInstanceThunk';
 export * from './hooks/useSelectorDeepComparison';
+export * from './selectors';
+export * from 'reselect';
```

Comment:
It's correct because initially I wanted to use more than only one function from reselect package, but it seems we are fine with just one so I will remove it.